docker/ckan/Dockerfile

```diff
@@ -111,19 +111,22 @@ RUN mkdir -p /var/log/supervisor /var/run/supervisor /etc/supervisor/conf.d
 RUN chown -R ckan:ckan /var/log/supervisor /var/log/ckan /var/run/supervisor /etc/supervisor
 RUN chown ckan:ckan /var/run
 RUN chmod gu+rw /var/run
-RUN chmod gu+s /usr/sbin/cron
+# RUN chmod gu+s /usr/sbin/cron
 COPY ./supervisord.conf /etc/supervisor/supervisord.conf
+COPY ./scheduler.py /usr/local/bin/scheduler.py
+RUN chmod +x /usr/local/bin/scheduler.py
+RUN $CKAN_VENV/bin/pip install schedule==1.1.0
 # Copy hello-cron file to the cron.d directory
-COPY crontab /etc/cron.d/crontab-ckan
+# COPY crontab /etc/cron.d/crontab-ckan
 
 # Give execution rights on the cron job
-RUN chmod 0644 /etc/cron.d/crontab-ckan
+# RUN chmod 0644 /etc/cron.d/crontab-ckan
 
 # Apply cron job
-RUN crontab -u ckan /etc/cron.d/crontab-ckan
+# RUN crontab -u ckan /etc/cron.d/crontab-ckan
 
 # Create the log file to be able to run tail
-RUN touch /var/log/cron.log
+# RUN touch /var/log/cron.log
```

Comment:
I guess that all the cron commented out stuff can be remoevd for good

Reply:
done